### DISCUSSION TITLE
COUNT (DISTINCT "colName") equivalent

### DISCUSSION THREAD

**Discussion by dakshshah96** [prisma/prisma#6094] [Q&A] [answered]
What would be the equivalent of this SQL query in Prisma where I'm trying to find the number of users who've ever made a payment:

```sql
SELECT COUNT(DISTINCT "userId") from payments;
```

Trying to use `distinct` with `count` gives me an error.

**Comment by ryands17** [comment, marked as answer]
@dakshshah96 👋 

Currently Prisma doesn't support `count` with `distinct`, but we have a request for the same https://github.com/prisma/prisma/issues/4228. So do follow this request and add a 👍 so that we can set the priority.

As a workaround, you can use the above raw query [directly](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
